app/uk/gov/hmrc/digitalservicestax/journeys/ReturnJourney.scala

```diff
@@ -32,6 +32,9 @@ object ReturnJourney {
   type ReturnTellTypes = Confirmation[Return] :: CYA[Return] :: GroupCompany :: NilTypes
   type ReturnAskTypes = (NonEmptyString, Boolean) :: DomesticBankAccount :: ForeignBankAccount :: Set[Activity] :: Money :: Percent :: Boolean :: List[GroupCompany] :: NilTypes
 
+  private def message(key: String, args: String*) =
+    Map(key -> Tuple2(key, args.toList))
```

Comment:
It's not immediately clear what this function is used for, maybe move it into askAmountForCompanies if you're only planning to use it there? 

Also `Tuple2(a,b)` is the same thing as `(a,b)`. Is this an IntelliJism?